engineering_tools/opendrive_lanelet2_converter/README.md

```diff
@@ -0,0 +1,25 @@
+# opendrive2lanlet2
+
+opendrive2lanlet2 is a Python script for converting opendrive maps to lanelet2 map readable by autoware.
```

Comment:
I believe this script only converts the 2d information from the map (elevation is dropped) Please add a node about this somewhere in the readme